common/src/main/java/com/sequenceiq/cloudbreak/logger/PropertyHelper.java

```diff
@@ -0,0 +1,31 @@
+package com.sequenceiq.cloudbreak.logger;
+
+import org.apache.commons.lang3.StringUtils;
+
+public class PropertyHelper {
```

Comment:
I would rename it to indicates that this class holds the logger related configurations that comes either from property or not. For example LoggerConfiguration.java